iron-overlay-backdrop.html

```diff
@@ -118,10 +118,14 @@
       this.style.zIndex = this._manager.backdropZ();
       // close only if no element with backdrop is left
       if (this._manager.getBackdrops().length === 0) {
+        // Read style before setting opened.
+        var cs = getComputedStyle(this);
+        var noAnimation = (cs.transitionDuration === '0s' || cs.opacity == 0);
```

Comment:
Exactly, because no transition will happen :)
Here we are in the `close()` method, which should go from `opacity: 1` to `opacity: 0`. If already 0, don't even bother waiting!